network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/tck/extensions/SecondaryVoltageControlTest.java

```diff
@@ -16,17 +16,8 @@
 public class SecondaryVoltageControlTest extends AbstractSecondaryVoltageControlTest {
 
     @Test
-    public void pilotPointTargetVoltageNotificationTest() {
-        // notify is not implemented in powsybl-network-store-iidm-impl
-    }
-
-    @Test
-    public void controlUnitParticipateNotificationTest() {
-        // notify is not implemented in powsybl-network-store-iidm-impl
-    }
-
-    @Test
-    public void extensionRemovalAndCreationNotificationTest() {
-        // notify is not implemented in powsybl-network-store-iidm-impl
+    public void variantTest() {
+        // https://github.com/powsybl/powsybl-core/pull/3454 manage variant for PilotPointImpl
+        // it should be implemented
```

Comment:
to check when we do it and how @etiennehomer

Reply:
Yes, we can do this after the powsybl upgrade. I create a ticket